jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
Currently there is no necessity of putting these functions in util.l, because there is no other dual arm tasks. (When we have, it is time to move these methods to baxter-robot or baxter-interface of `baxtereus`)
Both `get-opposite-arm` and `get-arm-controller` are required for arc task (because it demands dual-arm parallel tasking),
so it is ok to put these functions in `arc-interface`.

Current best solution IMO is:

- `get-opposite-arm` -> `jsk_arc2017_baxter::baxter-robot`
- `get-arm-controller` -> `jsk_arc2017_baxter::baxter-interface`